docs/dev-maintenance.md

```diff
@@ -1,21 +1,11 @@
-# Regular developer tasks dev-maintenance
-
-?> *TODO* update the content to work with modules in `terraform-ibm-modules` in GitHub.
+# Regular developer tasks :id=dev-maintenance
```

Comment:
Do you know whether it is possible to hide the meta-data (the :id=dev-maintenance part) when the content is rendered as regular markup. Perhaps by putting the meta-data in comments (<!-- ... -->)?